modules/rules_erlang/2.4.0/MODULE.bazel

```diff
@@ -0,0 +1,15 @@
+module(
+    name = "rules_erlang",
+    version = "2.4.0",
+    compatibility_level = 2,
+)
+
+download_xrefr = use_extension(
+    "@rules_erlang//bzlmod:extensions.bzl",
```

Comment:
When referring to your own repo, it's better to not include the `@<repo name>` part.

Reply:
I'm happy to change it, though that would make the file out of sync with what's in the `rules_erlang` repo. I presume it's better to just fix this in the next `rules_erlang` release?

Reply:
Yes, but it the one checked in the BCR will always override the one in the source archive. So it's OK to update here first.